apps/client/Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN yarn build:all
 
 # Create image for the app by copying build artifacts from builder
 FROM nginx:stable-alpine as runner
+RUN rm /etc/nginx/conf.d/default.conf
```

Comment:
A fix in Dockerfile for FE projects
Not purely related to this PR, but found during testing docker flow